test/test_bpmn_cwp.py

```diff
@@ -1,42 +1,413 @@
 # type: ignore
-from bpmncwpverify.visitors import PromelaGenVisitor
-from bpmncwpverify.cwp import Cwp
+from bpmncwpverify.visitors import CwpLtlVisitor, PromelaGenVisitor
+from bpmncwpverify.cwp import Cwp, CwpEdge, CwpState
 from bpmncwpverify.state import SymbolTable
 import inspect
 from returns.pipeline import is_successful
-from unittest.mock import MagicMock
+from unittest.mock import MagicMock, patch, call
```

Comment:
Let's use[ pytest-mock](https://pypi.org/project/pytest-mock/) rather than `MagicMock` since I `pytest-mock` is meant for `pytest`.